xml/art_sle_ha_pmremote.xml

```diff
@@ -73,20 +85,17 @@
   </itemizedlist>
  </sect1>
 
-  <sect1 xml:id="sec.ha.pmremote.concept">
+ <sect1 xml:id="sec.ha.pmremote.concept">
    <title>Conceptual Overview and Terminology</title>
-   <!-- Add a picture? -->
    <para>
     A regular cluster may contain up to 32 nodes. With the &pmremote; service,
     &ha; clusters can be extended to include additional nodes beyond this
-    limit by using guest nodes.
-    <remark>toms 2017-04-20:  yangao: think this phrase assumes the scenario
-     "baremetals as remote nodes" has been introduced previously. But
-     currently the documentation covers only the scenario of "guests as
-     remote nodes". We will probably either need to introduce "baremetals as
-     remote nodes" first or rephrase this.
-     Acutally the only use case I've seen from our customers so far is
-     "baremetals as remote nodes" in SUSE cloud.</remark>
+    limit.
+   </para>
+   <para>
+    The &pmremote; service can be operated as a physical node or as a virtual
+    guest node. The physical &pmremote; is usually used for &cloud;.
+    <remark>toms 2017-07-11: Any examples for use case 2?</remark>
     Unlike remote nodes, a guest node is a
     resource managed by the cluster. As such, they are not bound to the
```

Comment:
Unlike cluster nodes, either type of  remote node is managed as a resource by the cluster.